openlibrary/catalog/merge/test_merge_marc.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
Py2 is now gone.  That line will get auto stripped by `pyupgrade` so I will avoid doing a bunch of manual edits for this legacy Py2 clutter.  The line has no effect in Py3 because all .py files are assumed to be utf-8.

Reply:
I reviewed these lines because they new additions in this PR, not legacy Py2 clutter AFAICT -- unless the Github diffing is doing something odd?